packages/lodestar/src/cli/commands/eth1-private-network.ts

```diff
@@ -45,10 +47,23 @@ export class Eth1PrivateNetworkCommand implements CliCommand {
       });
   }
 
-  public async action(options: IEth1CommandOptions, logger: ILogger): Promise<PrivateEth1Network> {
+  public async action(options: IEth1CommandOptions, logger): Promise<PrivateEth1Network> {
+    let loggingOptions: ILoggingOptions;
     if (options.loggingLevel) {
-      logger.setLogLevel(LogLevel[options.loggingLevel]);
+      loggingOptions = {
+        loggingLevel: LogLevel[options.loggingLevel],
```

Comment:
you can collapse the if statement into one

`loggingLevel: LogLevel[options.loggingLevel] || LogLevel.DEFAULT,`

Reply:
this applies to all similar instances